docs/sources/panels-visualizations/panel-editor-overview/index.md

```diff
@@ -36,7 +36,7 @@ This section describes the areas of the Grafana panel editor.
 
 1. Visualization preview: The visualization preview section contains the following options:
 
-   - **Table view:** Convert any visualization to a table so that you can see the data. Table views are useful for troubleshooting.
+   - **Table view:** Convert any visualization to a table so that you can see the data. Table views are useful for troubleshooting. This view only contains the raw data. It does not include transformation you might have applied to the data.
```

Comment:
```suggestion
   - **Table view:** Convert any visualization to a table so that you can see the data. Table views are useful for troubleshooting. This view only contains the raw data. It does not include transformations you may have applied to the data, nor does it include all the formatting options available in the [Table Panel Visualization](https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/table/).
```

or something like that maybe?